src/pages/memberhsip/helpers/convertBooleantoString.ts

```diff
@@ -0,0 +1,8 @@
+export default function convertBooleanToString(
```

Comment:
Could you add some basic tests for this?

Reply:
Added tests for all existing helper functions.